pki/src/test/java/org/hyperledger/besu/pki/keystore/SoftwareKeyStoreWrapperTest.java

```diff
@@ -158,7 +159,7 @@ public void getCertificateChain() throws Exception {
     when(keyStore.getCertificateChain(CERTIFICATE_ALIAS))
         .thenReturn(new Certificate[] {certificate});
 
-    assertThat(keyStoreWrapper.getCertificateChain(CERTIFICATE_ALIAS)).hasSize(1);
+    assertEquals(keyStoreWrapper.getCertificateChain(CERTIFICATE_ALIAS), 1);
```

Comment:
This is comparing an array of Certificate with an int. The original assertion using hasSize is the correct thing to do

Reply:
fixed